sdk/cloudmachine/Azure.Provisioning.CloudMachine/src/FeaturesExtensions/AIFoundry.cs

```diff
@@ -0,0 +1,185 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.Projects;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Primitives;
+using Microsoft.Azure.Amqp.Framing;
+using System.Collections.Generic;
+
+namespace Azure.Provisioning.AIFoundry;
+
+internal class AIFoundryCognitiveServiceCdk : NamedProvisionableConstruct
+{
+    public AIFoundryCognitiveServiceCdk(string bicepIdentifier, string name)
+        : base(bicepIdentifier)
+    {
+        Name = name;
+    }
+
+    public string Name { get; set; }
+    protected override IEnumerable<BicepStatement> Compile()
+    {
+        ResourceStatement cs = new(
+            BicepIdentifier,
+            new StringLiteralExpression("Microsoft.CognitiveServices/accounts@2021-10-01"),
+                new ObjectExpression(
+                new PropertyExpression("name", Name),
+                new PropertyExpression("location", new IdentifierExpression("location")),
+                new PropertyExpression("sku",
+                    new ObjectExpression(
+                        new PropertyExpression("name", "S0")
+                    )
+                ),
+                new PropertyExpression("kind", "AIServices"),
+                new PropertyExpression("identity",
+                    new ObjectExpression(
+                        new PropertyExpression("type", "SystemAssigned")
```

Comment:
can we utilize our user-assigned CM identity here?

Reply:
we decided to do as a separate PR